storage/cql_iface/tests/mockIStorage.py

```diff
@@ -32,7 +32,7 @@ def setID(self, st_id):
 
     storage_id = property(getID, setID)
 
-    def __new__(cls, *args, name='', **kwargs):
+    def __new__(cls, name='', *args, **kwargs):
```

Comment:
In this way, name is a kwarg, then there are the args, and then the kwargs. This can cause problems, args should be groupped on the left, and kwargs groupped on the right:
```python
In [13]: class A(): 
    ...:     def do(self, name='', *args, **kwargs): 
    ...:         if name in args: 
    ...:             print('found')      
    ...:                                                                                                                                                                                      

In [14]: a = A()                                                                                                                                                                              

In [15]: a.do(1,2,name=3,some=4)  
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-15-41f1b9f30790> in <module>
----> 1 a.do(1,2,name=3,some=4)

TypeError: do() got multiple values for argument 'name'

```